### PR TITLE
RDM-11090: Postgress v11 upgrade

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-11090

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by danlysiak — APPROVED** [review]

**Review by Thor-tech-of-metal — APPROVED** [review]
LGTM

File: k8s/ithc/common/ccd/definition-store-api.yaml [hunk 1]
```diff
@@ -19,9 +19,22 @@ spec:
...
+            - ccd-ELASTIC-SEARCH-URL
+            - ccd-ELASTIC-SEARCH-PASSWORD
+            - storage-account-primary-connection-string
+            - AppInsightsInstrumentationKey
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/definition-store-api:pr-974-398e4709
       environment:
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-ithc.postgres.database.azure.com
+        DEFINITION_STORE_DB_NAME: ccd_definition_store
+        DEFINITION_STORE_DB_USERNAME: ccd@ccd-definition-store-api-postgres-db-v11-ithc
```
Comment (Thor-tech-of-metal) [inline k8s/ithc/common/ccd/definition-store-api.yaml:37]: One min, check in azure if it is the generated DEFINITION_STORE_DB_NAME VALUE for version 11.  "ccd_definition_store" . Just to check

**Review by Thor-tech-of-metal — APPROVED** [review]

File: k8s/ithc/common/ccd/definition-store-api.yaml [hunk 2]
```diff
@@ -19,9 +19,22 @@ spec:
...
+            - ccd-definition-s2s-secret
+            - ccd-ELASTIC-SEARCH-URL
+            - ccd-ELASTIC-SEARCH-PASSWORD
+            - storage-account-primary-connection-string
+            - AppInsightsInstrumentationKey
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/definition-store-api:pr-974-398e4709
       environment:
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-ithc.postgres.database.azure.com
+        DEFINITION_STORE_DB_NAME: ccd_definition_store
```
Comment (smathangi) [inline k8s/ithc/common/ccd/definition-store-api.yaml:36]: don't need these two `DEFINITION_STORE_DB_NAME`  (same as base chart value) `DEFINITION_STORE_DB_USERNAME` (gets from secrets / boostrap.yaml)

File: k8s/ithc/common/ccd/definition-store-api.yaml [hunk 3]
```diff
@@ -19,9 +19,22 @@ spec:
   values:
     java:
       replicas: 2
+      keyVaults:
+        ccd:
+          secrets:
+            - definition-store-api-POSTGRES-USER-V11
```
Comment (smathangi) [inline k8s/ithc/common/ccd/definition-store-api.yaml:25]: Does pr-974 tag has changes in bootstrap.yaml?

File: k8s/ithc/common/ccd/definition-store-api.yaml (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply k8s/ithc/common/ccd/definition-store-api.yaml:37]: copied these from azure

File: k8s/ithc/common/ccd/definition-store-api.yaml (hunk 3 above)
Comment (kiran-yenigala-hmcts) [reply k8s/ithc/common/ccd/definition-store-api.yaml:25]: Yes, it has

**Review by smathangi — APPROVED** [review]

**Pull request merged**